src/components/shared/MarkerTooltipContents.js

```diff
@@ -781,6 +804,12 @@ type OwnProps = {|
   +marker: TracingMarker,
   +threadIndex: ThreadIndex,
   +className?: string,
+  +markerMeta: {
```

Comment:
```suggestion
  +markerMeta?: {
```

This is requiring network information for every single marker, which doesn't really make sense. I believe this should be marked as optional, and is the source of the remaining Flow errors.